docs/tendermint-core/light-client.md

```diff
@@ -30,11 +31,26 @@ known validator set. From there, all the application state is verifiable with
   name-registry without worrying about fork censorship attacks, without posting
   a commit and waiting for confirmations. It's fast, secure, and free!
 
-## Where to obtain trusted height & hash
+## Security
+
+A light client is initialized from a point of trust using [Trust Options](https://pkg.go.dev/github.com/tendermint/tendermint/light?tab=doc#TrustOptions),
+a provider and a set of witnesses. This sets the trust period: the period that
+full nodes should be accountable for faulty behavior and a trust level: the
+fraction of validators in a validator set with which we trust that at least one
+is correct. As Tendermint consensus can withstand 1/3 byzantine faults, this is
+the default trust level, however, for greater security you can increase it (max:  
+1).
+
+Similar to a full node, light clients can also be subject to byzantine attacks.
+A light client also runs a detector process which cross verifies headers from a
+primary with witnesses. Therefore light clients should be set with enough witnesses.
 
-[Trust Options](https://pkg.go.dev/github.com/tendermint/tendermint/light?tab=doc#TrustOptions)
+If the detector observes a faulty provider it will report it to another provider
```

Comment:
I think we should state that the light client is not **safe** when a) more than {trust_level} of validators are malicious b) all witnesses are malicious

Reply:
Sure, this would be in replacement of lines 44 - 49 right? or the entire section

Reply:
not a replacement, but rather an addition which clearly states the fact.